src/guide/get-started/operate-iroha-via-cli.md

```diff
@@ -0,0 +1,359 @@
+::: tip
+
+To customize the generated keys, you can specify a number of parameters. For instance, `kagami` can use one of four available algorithms to generate cryptographic keys.
+
+To learn more about generating cryptographic keys with `kagami`, available algorithms, and other parameters, see [Generating Cryptographic Keys with Kagami](/guide/security/generating-cryptographic-keys.md#kagami)
+
+:::
+
+For the purposes of this tutorial, we will use the following key pair:
+
+```bash
+Public key (multihash): "ed0120ABA0446CFBD4E12627FFA870FB37993ED83EB1AE0588184B90D832A64C24C379"
+Private key (multihash): "802620CBD3D701B561FE98463767729176404DC757D690F78980B8FDD40C171CCB8EB5"
+```
+
+To register a new account within the _looking_glass_ domain, run:
+
+```bash
+$ iroha account register --id="ed0120ABA0446CFBD4E12627FFA870FB37993ED83EB1AE0588184B90D832A64C24C379@looking_glass"
+```
+
+The `--id` argument in the above code snippet specifies the _account id_, the unique identifier of the account. It consists of the user public key (generated using `kagami`) and the domain it is associated with.
+
+If the account registration succeeds, you will receive a confirmation message. Like before, you should query the accounts to verify the details.
+
+```bash
+$ iroha account list all
+```
+
+::: details Expand to see the expected output
+
+```json
+[
+  {
+    "id": "ed0120E9F632D3034BAB6BB26D92AC8FD93EF878D9C5E69E01B61B4C47101884EE2F99@garden_of_live_flowers",
+    "metadata": {}
+  },
+  {
+    "id": "ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4@genesis",
+    "metadata": {}
+  },
+  {
+    "id": "ed0120ABA0446CFBD4E12627FFA870FB37993ED83EB1AE0588184B90D832A64C24C379@looking_glass",
+    "metadata": {}
+  },
+  {
+    "id": "ed012004FF5B81046DDCCF19E2E451C45DFB6F53759D4EB30FA2EFA807284D1CC33016@wonderland",
+    "metadata": {
+      "key": "value"
+    }
+  },
+  {
+    "id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland",
+    "metadata": {
+      "key": "value"
+    }
+  }
+]
+```
+:::
+
+## 5. Transfer a Domain
+
+We could change the keys and domain in `client.toml` at this point and continue working with the account we just created, but we wouldn't be able to do much in the _looking_glass_ domain, as our new account is not the owner of _looking_glass_, and cannot manage it.
+
+Let's transfer the _looking_glass_ domain to the account we created:
+
+1. Run the transfer command:
+
+  ```bash
+  $ iroha domain transfer --id="looking_glass" --from "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland" --to "ed0120ABA0446CFBD4E12627FFA870FB37993ED83EB1AE0588184B90D832A64C24C379@looking_glass"
+  ```
+2. Check that the ownership changed:
+
+  ```bash
+  $ iroha domain list all
+  ```
+
+3. Switch to the newly created account. For this, we need to modify the `public_key`, `private_key`, and `domain` in the `client.toml` config file with the ones we registered earlier.
```

Comment:
Maybe it should be clarified that the domain is not necessarily the same as the domain we just transferred, but the domain of the account we want to act as.

Reply:
fixed in commit fec265d9a98b1d356133521d2320837db466d62f